code/modules/recycling/conveyor2.dm

```diff
@@ -88,6 +90,10 @@
 /obj/machinery/conveyor/New(loc, newdir = null, building = 0)
 	. = ..(loc)
 
+	//Gotta go fast!
+	machines -= src
+	fast_machines += src

```

Comment:
put ALL the mining machines on this otherwise your belts will go fast but the machines will be hella slow FNR